src/player_factory.py

```diff
@@ -0,0 +1,29 @@
+from table import Table
+from wheel import Wheel
+from players.player import Player
+from players.martingale import Martingale
+from players.cancellation import PlayerCancellation
+from players.fibonacci import PlayerFibonacci
+from players.seven_reds import SevenReds
+from players.random import PlayerRandom
+from players.passenger57 import Passenger57
+from players.player1326.player1326 import Player1326
+
+
+def provide_player(player_name: str, table: Table, wheel: Wheel) -> Player:
```

Comment:
rename method to `player_factory` or `get_player`

Reply:
Done.